src/components/screens/new-questionnaire-screen.tsx

```diff
@@ -133,14 +166,18 @@ const NewQuestionnaireScreen = () => {
         }
       });
       closeAndClear();
-      navigate(-1)
+      navigate(-1);
       return createdQuestionnaire;
     } catch (error) {
       setError(`${strings.error.questionnaireSaveFailed}, ${error}`);
-      setLoading(false);
     }
+    setLoading(false);
   };
 
+  useEffect(() => {
+    updateTooltipAndIsDisabled();
+  }, [questionnaire.title, questionnaire.description]);
```

Comment:
Rather than calling this in this UE and in the handleQuestionnaireInputChange, could it not be called directly from (and only from) the tooltip e.g. 
<Tooltip title={renderToolTip()}

This should work fine, you can see a very similar implementation here if it helps:
https://github.com/Metatavu/lipsanen-project-management-ui/pull/82/files